docker-compose-generator/docker-fragments/opt-add-balanceofsatoshis.yml

```
@@ -0,0 +1,14 @@
+version: "3"
+services:
+  bos:
+    container_name: bos
+    image: "alexbosworth/balanceofsatoshis:latest"
+    restart: unless-stopped
```

```suggestion
    restart: none
```

It's just a script, the container would restart endlessly otherwise.